test/PowerNotifications/PowerTests.cpp

```diff
@@ -0,0 +1,205 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "winrt/Microsoft.ProjectReunion.h"
+
+using PowerEventHandle =
+winrt::Windows::Foundation::EventHandler<winrt::Windows::Foundation::IInspectable>;
+
+using namespace winrt::Microsoft::ProjectReunion;
+
+namespace ProjectReunionPowerTests
+{
+    constexpr auto TIMEOUT = 2000;
```

Comment:
Why 2000?

What's the expected average and worst case timeouts you'll encounter?

Reply:
TBH I just choose a value where I was sure that it will succeed. Informal testing shows they take <500 ms and the Power team also didn't have data on this. Do you have any suggestions?

Reply:
NIT: c_TimeoutInMs

Reply:
Or, change this to a std::chrono::duration.  Makes it more explicit with what you mean.

Reply:
The variable name has been changed. I didn't use chrono as it would need another level of conversion and the variable name makes it clear. 
Will take suggestions on a possible better time out value though. :)